IBSWeb/wwwroot/css/modern-navbar.css

```
@@ -0,0 +1,943 @@
+    color: var(--mnav-on-primary);
+    white-space: normal;
+    overflow-wrap: anywhere;
+}
+
+.mm-item .mnav-col-item-sub,
+.mm-trigger .mnav-col-item-sub {
+    font-size: 11px;
+    color: var(--mnav-on-primary-70);
+    margin-top: 1px;
+    white-space: normal;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)